helper/mstats/mstats.go

```diff
@@ -0,0 +1,45 @@
+package mstats
```

Comment:
That's not very important, but it might be overall better (at least to add a TODO here) that this should be migrated to https://pkg.go.dev/runtime/metrics

Reply:
Can be removed - some metrics (gc/cache hits-miss/query stats) planed in future